apps/files/src/components/FileActions.vue

```diff
@@ -30,6 +36,21 @@ export default {
       isOpen: false
     }
   },
+  computed: {
+    $_label () {
+      const translated = this.$gettext('Open %{fileName} in')
+      return this.$gettextInterpolate(translated, { fileName: this.filename })
+    },
+
+    $_closeActionsButtonLabel () {
+      return this.$gettext('Close actions menu')
```

Comment:
@marcus-herrmann what would be the best text here as aria label? THX

Reply:
@DeepDiver1975 "Show options", "Show file actions" or the like.

Reply:
@DeepDiver1975 "Close file actions menu" should be fine.

Reply:
@LukasHirt ^

Reply:
Adjusted :wink: